### ISSUE TITLE
 Introduce a setting to automatically burn secrets after a specified number of unsuccessful attempts.

### ISSUE THREAD

**Issue by vdekercd** [onetimesecret/onetimesecret#297]
**Current behavior**

We are restricted to five attempts per machine. This means that on my personal device, I am prevented from making further password attempts after five errors, even if I try using incognito mode across multiple browsers. However, I am able to retry on another machine connected to the same Wi-Fi network, such as my smartphone.

**Desired behavior**

If the number of unsuccessful attempts exceeds a predetermined threshold, regardless of the originating machine, the secret will be automatically burned if the "BurnAfterXAttempts" setting is configured with a value greater than zero.

**Comment by delano** [comment]
Yeah that's a great idea. I've added this to our redesign project and we'll follow-up as part of that work.
